<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportData</class>
 <widget class="QDialog" name="ExportData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <item>
      <widget class="QLabel" name="label_groupNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>组号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_groupNumber">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>一组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>二组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>三组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>四组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>五组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>六组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>七组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>八组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>九组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十一组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十二组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十三组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十四组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十五组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十六组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十七组</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>年龄</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <widget class="QLabel" name="label_29">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>独生子女</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_onlyChild">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QLabel" name="label_27">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>子女数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_childrenAmount">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>婚姻状况</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_maritalStatus">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未婚</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>已婚</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>离婚</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>再婚</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其他</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <widget class="QLabel" name="label_32">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>优抚对象</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_entitledGroup">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_25">
     <item>
      <widget class="QLabel" name="label_53">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>享受低保</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_subsistenceAllowance">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <widget class="QLabel" name="label_58">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>迁入人员</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_immigration">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_29">
     <item>
      <widget class="QLabel" name="label_59">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>迁出人员</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_emigration">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_30">
     <item>
      <widget class="QLabel" name="label_60">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>就业人员</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_occupation">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_28">
     <item>
      <widget class="QLabel" name="label_54">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>退休</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_retirement">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_26">
     <item>
      <widget class="QLabel" name="label_57">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>党员</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_partyMember">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>不确定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pBtn_enquiry">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_return">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
